### PR TITLE
story 2310: added use of query parameters in ReviewPeriods.jsx

### PR DESCRIPTION
**Pull request by mvolkmann**

### CODE REVIEW

File: web-ui/src/components/reviews/periods/ReviewPeriods.jsx [hunk 1]
```diff
@@ -6,28 +6,32 @@ import DeleteIcon from '@mui/icons-material/Delete';
 import UnarchiveIcon from '@mui/icons-material/Unarchive';
 import WorkIcon from '@mui/icons-material/Work';
 
-import Box from '@mui/material/Box';
```
Comment (mvolkmann) [inline web-ui/src/components/reviews/periods/ReviewPeriods.jsx:9]: I combined these imports into one.

File: web-ui/src/components/reviews/periods/ReviewPeriods.jsx [hunk 2]
```diff
@@ -135,6 +139,20 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
   const userProfile = selectUserProfile(state);
   const isAdmin = userProfile?.role?.includes('ADMIN');
 
+  useQueryParameters([
+    {
+      name: 'add',
```
Comment (mvolkmann) [inline web-ui/src/components/reviews/periods/ReviewPeriods.jsx:144]: The only state I see that we can track with query parameters on this page is whether the "ADD REVIEW PERIOD" button was clicked.

File: web-ui/src/components/reviews/periods/ReviewPeriods.jsx [hunk 3]
```diff
@@ -446,7 +465,8 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
                     key={`period-lit-${id}`}
                     onClick={() => onPeriodClick(id)}
                     primary={
-                      name + (reviewStatus === ReviewStatus.OPEN ? ' - Open' : '')
```
Comment (mvolkmann) [inline web-ui/src/components/reviews/periods/ReviewPeriods.jsx:449]: Prettier did this.

**Review by jackkeller — APPROVED** [review]
Approved! 🚢

**Review by vhscom — APPROVED** [review]

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**